modules/dicom_archive/php/viewdetails.class.inc

```diff
@@ -56,7 +57,8 @@ class ViewDetails extends \NDB_Form
         $this->DB         = \Database::singleton();
         $this->tarchiveID = $_REQUEST['tarchiveID'] ?? '';
 
-        if ((!empty($_REQUEST['tarchiveID'])) && ($this->_hasAccess())) {
+        $user = User::singleton();
```

Comment:
Namespace on User?

Reply:
yeah, I had some conflicts when rebasing it, I must have not resolved them properly.